.github/CODEOWNERS

```diff
@@ -66,7 +66,6 @@
 
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28032&commitId=11fc744c5d218be5e1910995d2fb5a036d8fe264&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1046 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PeoplePicker.Normal.suggestions.chromium.png|12| Changed|

</details>


<div id="vrtCommentfluentuiv8"/>